Gruntfile.js

```diff
@@ -7,7 +7,7 @@ module.exports = function(grunt) {
 See more about the WaterColorBot @ http://watercolorbot.com or\n\
 Fork & improve the project @ https://github.com/evil-mad/robopaint";
   var appTags = 'GNOME;GTK;Arts;SVG';
-  var version = '2.0.0-Beta.1'; // TODO: Load this from the package file
+  var version = '2.0.0-Beta.2'; // TODO: Load this from the package file
```

Comment:
Feh. Just load it from the package file and be done with it. Yes, it's more complex, but it means we'll never have to fiddle with it again. **PROTIP:** If you directly `require()` a json file, it'll give you back its object!